File: liquibase/src/main/java/com/omidmohebbise/liquibase/repository/PersonJpaRepository.java
package com.omidmohebbise.liquibase.repository;


import com.omidmohebbise.liquibase.entity.Person;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.stereotype.Repository;

@Repository
public interface PersonJpaRepository extends JpaRepository<Person,Long>, JpaSpecificationExecutor {

}

File: kotlin/first-app-kptolin/src/main/kotlin/com/omidmohebbise/firstappkt/firstappkotlin/repository/PersonJpaRepository.kt
package com.omidmohebbise.firstappkt.firstappkotlin.repository

import com.omidmohebbise.firstappkt.firstappkotlin.entity.PersonEO
import org.springframework.data.repository.CrudRepository

interface PersonJpaRepository : CrudRepository<PersonEO, Long> {
}